Sources/_Helpers/AssertSnapshot+Platform.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import SnapshotTesting
+import XCTest
+
+/// A platform aware version of the standard `assertSnapshot`
+/// This is used to point to different subdirectories of the `__Snapshots__`
+/// directory. On different platforms, the different textual representations
+/// are different for a number of reasons, so this gives us a way to differentiate
+/// where needed.
+public func platformSpecificAssertSnapshot<Value, Format>(
```

Comment:
I understood the issue with the default implementation on the `assertSnapshot` method, but I'd prefer to open a discussion on the `swift-snapshot-testing` repository to check if there is any other solution to this.

Reply:
Issue re-opened on the snapshot testing repo https://github.com/pointfreeco/swift-snapshot-testing/issues/815

Reply:
I ended up fixing this so it's no longer needed!